packages/ssx-react/package.json

```diff
@@ -4,6 +4,33 @@
   "description": "SSX React Hooks",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
+  "exports": {
```

Comment:
We should look at building and exporting different module types as well, in all of our JS packages.